### DISCUSSION TITLE
请问配置里的"n_cookie": "",是什么

### DISCUSSION THREAD

**Discussion by mc-doge666** [CarterGunale/MusicBot#20] [Q&A] [answered]
请问配置里的"n_cookie": "",具体填的是什么

**Comment by ddxgtx** [comment, marked as answer]
网易云音乐的cookie
可以参考这篇文章：https://blog.tangbao.ltd/p-8.html/

**Reply by Vanillekz** [reply]
想问一下，我部署完成后，机器人可以进频道但是不能自动播放，看到报错代码zmq:tcp://127.0.0.1:1234:protocol not found，请问这个怎么解决。

**Reply by ddxgtx** [reply]
你看下你的config.json
里面的这一段代码  "schedule_server_address": "http://localhost/",没有以http://开头，或者出现空格等格式问题：

**Reply by Vanillekz** [reply]
你运行的时候会有voiceAPI这个screen吗

**Reply by ddxgtx** [reply]
> 你运行的时候会有voiceAPI这个screen吗

我是下面五个screen
scheduler.py
MiguMusicApi
​QQMusicApi
NeteaseCloudMusicApi
​core.py

**Reply by Vanillekz** [reply]
好吧 我的进语音只能手动点卡片播放，不能自动播放。你的可以自动播放吗

**Reply by ddxgtx** [reply]
可以

**Reply by Vanillekz** [reply]
好吧 可能是我系统问题，你是只改了config，然后用的这个评论的run.sh是吧

**Reply by ddxgtx** [reply]
是的

**Reply by Vanillekz** [reply]
想问一下 服务器端怎么获取cookie 没有UI

**Reply by ddxgtx** [reply]
> 想问一下 服务器端怎么获取cookie 没有UI

网页端获取cookie后粘贴到config然后把config上传到服务器不就行了吗

**Reply by Vanillekz** [reply]
不同IP的cookie可以共用吗。。。还是要用ssh远程连接

**Reply by ddxgtx** [reply]
> 不同IP的cookie可以共用吗。。。还是要用ssh远程连接

应该是可以的吧，我就是这样获取的cookie然后程序能正常的使用
